packages/webview_flutter/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -450,6 +469,33 @@ - (bool)loadUrl:(NSString*)url withHeaders:(NSDictionary<NSString*, NSString*>*)
   return true;
 }
 
+- (bool)postRequest:(NSDictionary<NSString*, id>*)request {
+  if (!request) {
+    return false;
+  }
+
+  NSString* url = request[@"url"];
+  if ([url isKindOfClass:[NSString class]]) {
+    id postData = request[@"postData"];
+    if ([postData isKindOfClass:[FlutterStandardTypedData class]]) {
```

Comment:
Please just use early returns for error handling instead of chaining methods for the sole purpose of reducing nesting.